shell/platform/android/io/flutter/embedding/android/FlutterFragment.java

```diff
@@ -358,8 +361,14 @@ public void onAttach(Context context) {
       // which means there shouldn't be any possibility for the Fragment Lifecycle to get out of
       // sync with the Activity. We use the Fragment's Lifecycle because it is possible that the
       // attached Activity is not a LifecycleOwner.
-      flutterEngine.getActivityControlSurface().attachToActivity(getActivity(), getLifecycle());
+      flutterEngine.getActivityControlSurface().attachToActivity(
+          getActivity(),
+          getLifecycle(),
+          platformViewsController
```

Comment:
if you own it, you should operate it too
e.g.

platformViewController.here'sTheActivity(getActivity())

Reply:
I don't think that makes much sense. This is typical dependency injection. Notice that `FlutterFragment` doesn't "operate" its `Activity` or `Lifecycle`, it passes those as references to the things that actually work with them. `PlatformViewsController` is no different.

Reply:
Talked offline, resolving old comments.